src/welcome/assets/components/QuickActionPanel.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { encodeCommandUri } from '../utils';
+
+export default class QuickActionPanel extends React.Component {
+    render() {
+        const actions = [
+            { name: "Create a New Project", command: "java.project.create" },
+            { name: "Open an Existing Project", command: "workbench.action.files.openFolder", os: "win" },
+            // { name: "Open an Existing Project", command: "workbench.action.files.openFileFolder", os: "mac" },
```

Comment:
Is it possible to check if the command `workbench.action.files.openFileFolder` is available or not and then decide which command to call?

Reply:
I'm also thinking of it. Looks like `data-os` is used in `<a>` nodes in "Overview" page. I also tried but it didn't work.